api/src/main/java/org/openmrs/hl7/HL7InQueueProcessor.java

```diff
@@ -56,8 +56,7 @@ public static void setCount(Integer count) {
 	public void processHL7InQueue(HL7InQueue hl7InQueue) {
 		
 		if (log.isDebugEnabled()) {
-			log.debug("Processing HL7 inbound queue (id=" + hl7InQueue.getHL7InQueueId() + ",key="
-			        + hl7InQueue.getHL7SourceKey() + ")");
+			log.debug("Processing HL7 inbound queue (id= {} ,Key= {})", hl7InQueue.getHL7InQueueId(), hl7InQueue.getHL7SourceKey());
```

Comment:
the `key` should be lowercase and you added some whitespaces.

so it should be
`log.debug("Processing HL7 inbound queue (id={},key={})",...`